micrometer-test/src/main/java/io/micrometer/core/util/internal/logging/MockLogger.java

```diff
@@ -28,25 +28,25 @@
 
 /**
  * Simple implementation of {@link InternalLogger} that does not produce any output or delegate the work to another logger,
- * instead it stores every log event in-memory so that the recorder log events can be fetched and verified by tests.
+ * instead it stores every log event in memory so that the recorded log events can be fetched and verified by tests.
  *
  * You should not create instances of this class directly, instead you can use the {@link InternalLoggerFactory} to get one.
  *
  * @author Jonatan Ivanov
  */
-public class InternalMockLogger implements InternalLogger {
+public class MockLogger implements InternalLogger {
     private final String name;
     private final Queue<LogEvent> logEvents = new ConcurrentLinkedQueue<>();
 
-    InternalMockLogger(String name) {
+    MockLogger(String name) {
         this.name = name;
     }
 
     /**
      * @return The recorded {@link LogEvent} instances, in descending order by age (the oldest is the first one).
      */
     public List<LogEvent> getLogEvents() {
-        return Arrays.asList(logEvents.toArray(new LogEvent[0]));
+        return new ArrayList<>(logEvents);
```

Comment:
The difference between the two is that as far as I remember the first one is immutable but I think we are good anyways.